### DISCUSSION TITLE
Specifying sourceIP ranges

### DISCUSSION THREAD

**Discussion by rajjaiswalsaumya** [strimzi/strimzi-kafka-operator#7468] [General]
How can we specify for external load balancer?
```
f:spec:
        f:externalTrafficPolicy: {}
        f:internalTrafficPolicy: {}
```

**Comment by scholzj** [comment]
I have no idea what the fidls above are. External load balancer is what Kubernetes create for you by default. when `type: loadbalancer` listener is created.

**Reply by rajjaiswalsaumya** [reply]
The above fields are to restrict access to the outer world and specify a NAT range only to allow access to.

**Reply by scholzj** [reply]
No, they are like they are copied from some metadata and not from any actual resource.
